File: 03 C C++ Python R and Mojo review/03-3 R Language Review/01 Pointers/06.R
# R equivalent for simulating matrix-like behavior using lists

arr1 <- c(1, 2, 3, 4)
arr2 <- c(5, 6, 7, 8)
matrix <- list(arr1, arr2)

for (row in matrix) {
  for (value in row) {
    cat(value, " ")
  }
  cat("\n")
}

File: 03 C C++ Python R and Mojo review/03-2 Python Review/01 Pointers/06.py
# Python equivalent for simulating matrix-like behavior using lists

arr1 = [1, 2, 3, 4]
arr2 = [5, 6, 7, 8]
matrix = [arr1, arr2]

for row in matrix:
    for i in range(len(row)):
        print(row[i], end=" ")
    print()